src/test/java/karate/sign_in/Sign_in.java

```diff
@@ -0,0 +1,10 @@
+package karate.sign_in;
+
+import com.intuit.karate.junit5.Karate;
+
+public class Sign_in {
+    @Karate.Test
+    Karate testSample() {
+        return Karate.run("classpath:karate/sign_in/sign_in.feature").relativeTo(getClass());
+    }
+}
```

Comment:
remember you don't need to specify the feature.. why don't you delete this classes? What are you using that for?